### ISSUE TITLE
Is it possible to add support for the loudspeakers in the Airpods pro 2 case?

### ISSUE THREAD

**Issue by Corlos2078** [d4rken-org/capod#39]

**Comment by d4rken** [comment]
No one knows how to do it :face_exhaling:.
If someone figures it out, I'd be happy to add it.

**Comment by K3NOXOFFICIAL** [comment]
Hasn't read and write operations been figured out by someone recently? Don't really know anything specific but might be a good idea to look into.

**Comment by xerootg** [comment]
This code popped up recently for setting airpods modes for windows, maybe that's a good starting point? https://github.com/bho3538/Airpods-Windows-Control

**Comment by d4rken** [comment]
Yes, the magicpods project has figured out read/write support. I could reproduce it on my desktop computer but not on Android. As far as I know, Android does not support the special Bluetooth connection type that this requires, at least i could not figure it out.

**Comment by d4rken** [comment]
Requires #215

**Comment by kavishdevar** [comment]
Actually, we don't need #215. As I've mentioned before in https://github.com/steam3d/MagicPodsCore/issues/14, the AirPods case host a GATT server. This is how the sound is played, and battery status is retrieved even when they're not connected. But, to tell it do something, we probably need to authenticate with it. Unfortunately, I haven't figured it out exactly out this "authentication" happens, and whatever I write to the characteristic is ignored by the case.
